### ISSUE TITLE
[Feature Request] Ignore a specific todo

### ISSUE THREAD

**Issue by dlion** [delashum/obsidian-checklist-plugin#101]
Hi, 
I'm using the checklist-plugin combined with the calendar-plugin and I would like to "hide" or "ignore" an old check-list.

Right now I can fold the point but it's still there and I can't do anything about it.
I know that if I mark that todo with the X it will disappear but this is not what I wanted because I'm doing some statistics and it's going to bring to me "fake data".

![image](https://user-images.githubusercontent.com/2125236/143499699-7b319c14-2d46-455f-a3b3-15d4ab47918f.png)

I just wanted to get rid of it from the list.

Thank you

**Comment by delashum** [comment]
Hey @dlion I'm not exactly sure what you are looking to do here, would you mind explaining in a bit more detail? Are you looking to exclude certain files?

**Comment by dlion** [comment]
Hi @delashum 

despite would be awesome having a way to exclude more than one directory (right now, sadly I can exclude only one directory) I was thinking of a way to exclude from the checklist `old` todos.
For instance:
I create a todo today: `- [ ] Buy ice-cream` but for some reason, I don't do it and I just want to ignore it but I want to track it down anyway.
That entry will be always in the checklist and the only way to hide it is to fold the day in the list.
Is there (or would be possible to have) a way to mark a specific entry as #old or #ignored or something like that?

Right now, I came up with the solution to put a dash instead of a X inside the [ ] and apparently it works.

**Comment by delashum** [comment]
Yeah that's an interesting use case. Unfortunately your workaround won't work anymore in the next release as the plugin will now support any char inside the brackets as checked because that is how obsidian processes it 😬 i.e. obsidian considers `- [-]` as checked

(side note, the next release also includes a way to exclude multiple directories use glob pattern matching)

I can see a feature being added though where you can specify a tag that will "archive" todos

**Comment by dlion** [comment]
> Unfortunately your workaround won't work anymore

😥



> the next release also includes a way to exclude multiple directories use glob pattern matching

Great, do you know when it's going to be released?




> I can see a feature being added though where you can specify a tag that will "archive" todos

It would be very useful for my use-case if I haven't done something, marking it as done anyway just because I don't want to have it on my checklist sidebar seems quite wrong to me ahaha
Adding a specific hashtag to mark is as archived or old or ignored would be great, thanks!

**Comment by kerbless** [comment]
Merge with #118?
